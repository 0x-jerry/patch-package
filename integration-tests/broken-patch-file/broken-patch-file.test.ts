import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest("broken-patch-file", true)
