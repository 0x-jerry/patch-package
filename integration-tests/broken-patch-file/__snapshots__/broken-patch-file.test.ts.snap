// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test broken-patch-file: patch-package fails when patch file is invalid 1`] = `
"SNAPSHOT: patch-package fails when patch file is invalid

**ERROR** Failed to apply patch for package left-pad

  This error was caused because patch-package cannot apply the following patch file:

    patches/left-pad+1.1.3.patch

  If removing node_modules and trying again doesn't fix this, maybe there was
  an accidental change made to the patch file? If not, then it's probably a bug
  in patch-package, so please submit a bug report. Thanks!

    https://github.com/ds300/patch-package/issues


END SNAPSHOT"
`;
